src/makeClientFunctionWrapper.js

```diff
@@ -38,17 +47,21 @@ function makeClientFunctionWrapper({
     );
 
     return async t => {
+      const testName = t.testRun.test.name;
```

Comment:
I just want to verify that this is supported in all cases and all the TestCafe versions that we support

Reply:
Good idea. In the interim, we could make this more defensive (e.g., `t.testRun && t.testRun.test && t.testRun.test.name ? t.testRun.test.name || ''`